server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -106,6 +106,20 @@ public final class GitMirror extends AbstractMirror {
 
     private static final int GIT_TIMEOUT_SECS = 60;
 
+    @Nullable
+    private static Constructor<?> GIT_WITH_AUTH_CONSTRUCTOR;
+
+    static {
+        Constructor<?> constr = null;
+        try {
+            constr = Class.forName("com.linecorp.centraldogma.server" +
+                                   ".internal.mirror.Git6WithAuth")
+                                   .getDeclaredConstructor(GitMirror.class, File.class);
+        } catch (Exception e) {
+            logger.warn("Falling back to JGit 5:", e);
```

Comment:
I wonder if this shouldn't be a warning since this might be expected behavior for some users.

```suggestion
            logger.info("Falling back to JGit 5:", e);
```